src/app/themes.tsx

```diff
@@ -31,10 +32,12 @@ const lightTheme: CustomTheme = {
     primaryBackgroundColor: "#63a036",
     secondaryForegroundColor: "#ffffff",
     secondaryBackgroundColor: "#066940",
-    accentBackgroundColor: "#609fd3",
+    accentBackgroundColor: "#1b629c",
```

Comment:
why is this changed here? this should be in a new ticket (attached to the Themes VFB ticket number) if it's a themeing change - it's not part of 'creating a list page' necessarily!

Reply:
reverted